stable/kong/templates/deployment.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "{{ template "kong.fullname" . }}"
+  labels:
+    app: "{{ template "kong.fullname" . }}"
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+    {{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+    {{- end }}
+      labels:
+        app: "{{ template "kong.fullname" . }}"
```

Comment:
Please make sure you read our review guidelines and best practices.

Please change throughout the chart: 
`app: "{{ template "kong.name" . }}"`

Reply:
Sure @unguiculus, I'll update them. But I did notice `"{{ template "<chart>.fullname" . }}` being used in many of the charts.

Reply:
Current best practice is to use the name template.

Reply:
Thanks, I am in process of updating it.